Mathlib/CategoryTheory/Localization/SmallHom.lean

```diff
@@ -252,8 +252,7 @@ lemma equiv_smallHomMap (G : D₁ ⥤ D₂) (e : Φ.functor ⋙ L₂ ≅ L₁ 
     dsimp [γ]
     rw [liftNatTrans_app]
     dsimp
-    rw [id_comp, id_comp, comp_id]
-    rw [id_comp, comp_id]
+    rw [id_comp, id_comp, comp_id, id_comp, comp_id]
```

Comment:
Follow-up: would `simp only [id_comp]` work?

Reply:
We can even turn that whole subproof into a single `simp` call now!